bundles/binding/org.openhab.binding.knx/src/main/java/org/openhab/binding/knx/internal/dpt/KNXCoreTypeMapper.java

```diff
@@ -495,17 +805,19 @@ public Type toType(Datapoint datapoint, byte[] data) {
                 int g = Integer.parseInt(value.split(" ")[1].split(":")[1]);
                 int b = Integer.parseInt(value.split(" ")[2].split(":")[1]);
 
+                // return HSBType.fromRGB(r, g, b);
                 Color color = new Color(r, g, b);
                 return new HSBType(color);
             }
+
         } catch (KNXFormatException kfe) {
-            logger.info("Translator couldn't parse data for datapoint type '{}' (KNXFormatException).",
+            logger.info("Translator couldn't parse data for datapoint type ‘{}‘ (KNXFormatException).",
                     datapoint.getDPT());
         } catch (KNXIllegalArgumentException kiae) {
-            logger.info("Translator couldn't parse data for datapoint type '{}' (KNXIllegalArgumentException).",
+            logger.info("Translator couldn't parse data for datapoint type ‘{}‘ (KNXIllegalArgumentException).",
```

Comment:
Please restore the original single quotes here.

Reply:
done.